docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
       {
         name: 'disabled',
         type: 'boolean',
+        description: `Used to disable the entire SelectList. See the [states](#States) variant to learn more.`,
         defaultValue: 'false',
       },
       {
         name: 'errorMessage',
         type: 'string',
-        href: 'exampleWithError',
+        description: `Used to communicate error information to the user. Be sure to localize the text. See the [states](#States) variant to learn more.`,
       },
       {
         name: 'helperText',
         type: 'string',
-        description: 'More information about how to complete the form field',
-        href: 'helperText',
+        description: `Used to provide more information about the form field. Be sure to localize the text. See the [helper text](#Helper-text) variant to learn more.`,
       },
       {
         name: 'id',
         type: 'string',
+        description:
+          'A unique identifier to connect the underlying `<select>` with the associated label',
         required: true,
-        href: 'basicExample',
       },
       {
         name: 'label',
         type: 'string',
+        description: 'The label shown above the input. Be sure to localize the label.',
       },
       {
         name: 'name',
         type: 'string',
-        href: 'basicExample',
+        description: 'Used to specify the name of the control.',
```

Comment:
I had no context on this attribute, googled and found this

```
The name attribute specifies the name for a drop-down list.

The name attribute is used to reference elements in a JavaScript, or to reference form data after a form is submitted.

```

Any of the examples use it, so I wonder how we could explain this better. When to use it vs not.

Reply:
I don't think I have a good notion of how to describe this attribute. Looks like what we're using comes straight from MDN. I'd suggest we keep it for now.

Reply:
> Looks like what we're using comes straight from MDN

Guilty as charged. 😅 TBH I don't fully understand why we need this prop either. I'm fine with leaving it as-is for now and updating in the future

Reply:
Oh, no knock at all for using MDN! I think we go with this and re-evaluate if/when necessary.